Request for Review: RoR Association's Project (Private Events)

rloterh opened the pull request:
- Created a User controller with 'new', 'create',  and 'show' actions
- Added sessions to mange signed-in user's account.
- Created the Event and AttendEvent models and applied necessary associations and foreign keys.
- Users can create event and registered members can attend upcoming event.

krys2fa reviewed (changes requested):
## Nicely done @rloterh  👏 
Your app can be easily set up and works well. 👍 
Only a few changes required.
### Status: Changes Required ♻️ 

- [ ] [REQUIRED] Your app should have navigation links at the top of all pages ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/rails/associations/navigation_links_requirement.png) from [Odin](https://www.theodinproject.com/courses/ruby-on-rails/lessons/associations)).

- [ ] [REQUIRED] You should set up GitHub Actions on this repository so it will run tests on your Pull Request.

- [ ] [REQUIRED] Please ensure all your models have unit tests for validations and associations (Rspec) ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/rails/associations/unit_tests_requirement.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/49723079/))

- [ ] [REQUIRED] Please ensure authentication has integration tests (Rspec + Capybara) ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/rails/associations/integration_tests_requirement.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/49723079/))

- [ ] [REQUIRED] Please ensure event management features have integration tests (Rspec + Capybara) ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/rails/associations/integration_tests_requirement.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/49723079/))

Please request for another review after implementing the changes.

Happy Coding 🖥️  😃 

###### You can contact me on slack (Chris Amanor) for any clarification.

rloterh commented:
- Fix issues accordingly as recommended by first review.
- Added test cases for request, method and feature.
- Fixed linter issue.
- Added navigation elements.

rOluochKe reviewed (approved):
Hello @rloterh 
### You got it right :clap: The project has been approved. Congratulations :boom: :100:
### Merge and submit the project. Good luck with the next one.
- Happy coding!
